blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/TestIO.java

```diff
@@ -0,0 +1,24 @@
+package com.rackspacecloud.blueflood.io;
+
+/**
+ * Utility class to provide some test methods existing tests use.
+ */
+public interface TestIO {
```

Comment:
How about CassandraUtilsIO, it's specificly a utility class for doing Cassandra stuff, yes?